libs/dtm/dtm_recovery.py

```diff
@@ -271,3 +274,24 @@ def get_process_ids(health_obj, process):
         svc = switcher[process]
         fids = fids[svc]
         return True, fids
+
+    def perform_copy_objects(self, workload, que):
```

Comment:
IS existing lib in HA for copy object not usable?

Reply:
As per discussion, for now we can keep this as is and see if some chnages in existing lib will help here.
cc: @kulrk @ApurwaMohite